CMakeLists.txt

```diff
@@ -629,6 +629,38 @@ if (APPLE)
 	target_link_libraries(torrent-rasterbar PRIVATE "-framework CoreFoundation" "-framework SystemConfiguration")
 endif()
 
+# check if we need to link with libatomic (not needed on MSVC)
+if (NOT MSVC)
```

Comment:
`libatomic` is not a thing on windows. It seems like this check should really check for windows, not which compiler is being used (i.e. clang and mingw on windows probably shouldn't do this either)

Reply:
fixed.